.github/workflows/build.yml

```diff
@@ -10,10 +10,9 @@ jobs:
   build:
     strategy:
       matrix:
-        os: [ubuntu-latest]
         python: [3.6, 3.7, 3.8, 3.9]
```

Comment:
The "include" syntax looks interesting: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

Does something like that would work?

```
    strategy:
      matrix:
        os: [ubuntu-latest, macos-latest]
        include:
          - os: ubuntu-latest
            python: [3.6, 3.7, 3.8, 3.9]
```

Reply:
Let's try